### PR TITLE
fix(markdown): use large link and expressive lists

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

#6185 

### Description

Update markdown to use large or expressive variations for links and unordered/ordered lists

### CODE REVIEW

File: packages/web-components/src/components/markdown/markdown.ts [hunk 1]
```diff
@@ -77,6 +77,12 @@ class DDSMarkdown extends LitElement {
       this._hasRendered = true;
       render(lightDOMTemplateResult, this, { eventContext: this });
     }
+    this.querySelectorAll('bx-unordered-list').forEach(e => {
+      e.setAttribute('isexpressive', '');
+    });
+    this.querySelectorAll('bx-ordered-list').forEach(e => {
+      e.setAttribute('isexpressive', '');
+    });
```
Comment (kennylam) [inline packages/web-components/src/components/markdown/markdown.ts:85]: These selectors can be combined. Also, even though `isexpressive` is a string I think we should stick to the camel-casing (`isExpressive`).

```suggestion
    this.querySelectorAll('bx-ordered-list, bx-unordered-list').forEach(e => {
      e.setAttribute('isExpressive', '');
    });
```

**Review by kennylam — COMMENTED** [review]
Looks good @ariellalgilmore, just a small change.

**Review by kennylam — APPROVED** [review]
LGTM thanks @ariellalgilmore!

**Review by oliviaflory — APPROVED** [review]
Beautiful! 🌈

**Pull request merged**